### PR TITLE
feat(bitgo): add BitGo Korea entity

### PR DESCRIPTION
**Pull request by pranishnepal**
## What
Currently, "BiGo Sister Trust 1" represents BitGo Korea entity. This commit adds "BitGo Korea" as a new entity since that would be safer than changing the existing.

## Issue Number
TICKET: COPS-2076

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Testing
- Unit Tests 
- `yarn run unit-test --scope @bitgo/statics`

### CODE REVIEW

File: modules/statics/src/base.ts [hunk 1]
```diff
@@ -213,9 +213,14 @@ export enum CoinFeature {
    */
   CUSTODY_BITGO_SINGAPORE = 'custody-bitgo-singapore',
   /*
-   * This coin supports custody in BitGo Sister Trust 1 entities
+   * This coin supports custody in BitGo Sister Trust 1 entities.
+   * This used to represent BitGo Korea previously. Going forward, use BitGo Korea instead.
```
Comment (aarvay) [inline modules/statics/src/base.ts:217]:
```suggestion
   * @deprecated This used to represent BitGo Korea previously. Going forward, use `CUSTODY_BITGO_KOREA` instead.
```

Comment (pranishnepal) [reply modules/statics/src/base.ts]: updated, thanks!

File: yarn.lock
Comment (abhishekdhakla) [inline yarn.lock:1]: I think we don't need this one!

**Review by abhishekdhakla — CHANGES_REQUESTED** [review]

File: yarn.lock
Comment (pranishnepal) [reply yarn.lock:1]: i thought about it, but didn't see it in `.gitignore` so i let it commit. Shall i add it to `.gitignore,` then?

Comment (pranishnepal) [reply yarn.lock:1]: removed + added to `.gitignore`

**Review by abhishekdhakla — APPROVED** [review]

**Review by alebusse — APPROVED** [review]

**Review by lzjluzijie — APPROVED** [review]

**Pull request merged**